public/index.html

```diff
@@ -40,6 +40,197 @@
           <h1>Web Fundamentals: Web Systems</h1>
           <h2>Four Athens</h2>
         </section>
+          <section>
+              <div align="center">
+                    <img src="img/cta.gif"/>
+               </div>
+          <h1>Pt 1. Finding Your Call to Action</h1>
+        </section>
+           <section>
+          <h1>What is a 'Call To Action'? (CTA)</h1>
+               <br>
+               <p>What <em>action</em> do you want your visitors to your website to make?</p>
+        </section>
+           <section>
+          <h2>CTA Examples</h2>
+               <br>
+               <ul>
+                   <li>user fills out contact form</li>
```

Comment:
Again, not a requirement, but adding `class="fragment"` to the `<li>`s will let them roll in step by step.